internal/provider/ml_model/model.go

```diff
@@ -15,14 +16,28 @@ func NewVellumMLModelCreateRequest(ctx context.Context, mlModelModel *TfMLModelR
 	developedBy, _ := vellum.NewMlModelDeveloperFromString(mlModelModel.DevelopedBy.ValueString())
 	family, _ := vellum.NewMlModelFamilyFromString(mlModelModel.Family.ValueString())
 
-	// TODO: Pass in actual values rather than dummy keys
-	// Create an empty slice for features
-	features := []vellum.MlModelFeature{"CHAT_MESSAGE_SYSTEM"}
+	features := []vellum.MlModelFeature{}
+	for _, feature := range mlModelModel.ExecConfig.Features.Elements() {
+		feature, _ := vellum.NewMlModelFeatureFromString(feature.(types.String).ValueString())
+		features = append(features, feature)
+	}
+
+	metadata := map[string]interface{}{}
+	for key, tfvalue := range mlModelModel.ExecConfig.Metadata.Elements() {
+		value := tfvalue.(types.String).ValueString()
```

Comment:
For now I'm enforcing that all metadata is a mapping between string <> string

Reply:
[oos] I wouldn't be totally opposed to just enforcing this on Vellum's end too, now that we'll have consumers dependent on this fact